draft-rescorla-tls-rfc8446-bis.md

```diff
@@ -5194,10 +5199,10 @@ based on the expectations in this document, even when handling prior TLS version
 handshakes (see {{server-certificate-selection}}).
 
 TLS 1.2 and prior supported an "Extended Master Secret" {{RFC7627}} extension
-which digested large parts of the handshake transcript into the master secret.
-Because TLS 1.3 always hashes in the transcript up to the server Finished,
+which digested large parts of the handshake transcript into the secret and
+hence keys. Because TLS 1.3 always hashes in the transcript up to the server Finished,
 implementations which support both TLS 1.3 and earlier versions SHOULD
-indicate the use of the Extended Master Secret extension in their APIs
+indicate the use of the Extended Main Secret extension in their APIs
```

Comment:
```suggestion
indicate the use of the "Extended Master Secret" extension in their APIs
```

Reply:
I think we should leave this because that's actually the name of the extension in the spec. Perhaps we can work around it by not naming it?